docs/next/guides/integrate-with-angular/angular-custom-context-menu-example.md

```diff
@@ -76,3 +79,41 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 platformBrowserDynamic().bootstrapModule(AppModule).catch(err => { console.error(err) });
 ```
 :::
+
+## Related articles
+
+#### Related guides
+
+- [Adding comments via the context menu](@/guides/cell-features/comments.md#adding-comments-via-the-context-menu)
+- [Clipboard: Context menu](@/guides/cell-features/clipboard.md#context-menu)
+- [Context menu](@/guides/accessories-and-menus/context-menu.md)
+- [Icon pack](@/guides/accessories-and-menus/icon-pack.md)
+
+#### Related blog articles
+
+- [Customize Handsontable context menu](https://handsontable.com/blog/customize-handsontable-context-menu)
```

Comment:
Check with @AMBudnik if it's still valid.

Reply:
Ola says that for now it's OK (but they'll be double-checking blog posts soon).